backend/windmill-api/openapi.yaml

```diff
@@ -2086,6 +2085,36 @@ paths:
             application/json:
               schema: {}
 
+  /w/{workspace}/jobs/result_by_id/{flow_job_id}/{node_id}:
+    get:
+      summary: get job result by id
+      operationId: resultById
+      tags:
+        - job
+      parameters:
+        - $ref: "#/components/parameters/WorkspaceId"
+        - name: flow_job_id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: node_id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: skip_direct
+          description: TODO Add description, I'm not even really sure what this does
```

Comment:
Yeah my bad for poor naming but this flag skip checking the nodes as child of the flow of which we have the id for, useful in cases you already have done the check as we do inside the eval context of flows. 
As reminder, the algorithm look for result by id in every parent flows as we assume that it means they have been evaluated earlier and are in the direct path that led to the execution of the job that does this call.